File: 3.Presentation/ClientApp/app/models/playerDTO.ts
export class PlayerDTO{
    constructor(
        public name: string,
        public currentTeam: string,
        public league: string,
        public nationality: string,
        public position: string,
        public birthDay: string,
        public height: string,
        public weight: string,
        public strengths: string,
        public weaknesses: string,
        public description: string,
        public imageSource: string) { }
}

File: 1.Business/DTO/PlayerDTO.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace Business.DTO
{
    class PlayerDTO
    {
        [Required]
        public string Name { get; set; }

        [Required]
        public string League { get; set; }

        [Required]
        public string CurrentTeam { get; set; }

        public string Nationality { get; set; }

        public string Position { get; set; }

        public DateTime BirthDay { get; set; }

        public int Height { get; set; }

        public int Weight { get; set; }

        public string Strengths { get; set; }

        public string Weaknesses { get; set; }

        public string Description { get; set; }

        public string ImageName { get; set; }
    }
}
